src/components/BobaTile.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Image from 'next/image';
+
+interface BobaTileProps {
+  drinkName: string;
+  imageUrl: string;
+  price: number;
+  description: string;
+}
+
+export default function BobaTile({
+  drinkName,
+  imageUrl,
+  price,
+  description,
+}: BobaTileProps) {
+  return (
+    <article className="max-w-sm rounded overflow-hidden shadow-lg">
+      <Image
+        src={imageUrl}
+        width={234}
+        height={234}
+        className="w-full"
+        alt={drinkName}
+      />
+      <div className="px-3 pb-4 pt-4 bg-orange-100">
```

Comment:
if you want to match the OG site
```suggestion
      <div className="px-3 pb-4 pt-4 bg-[#F3ECE3]">
```